browser/admin/src/integrator/AdminIntegratorSettings.ts

```diff
@@ -66,6 +66,7 @@ const initTranslationStr = () => {
 class SettingIframe {
 	private wordbook;
 	private xcuEditor;
+	private _browserSetting;
```

Comment:
Can we rename browser/_browser to view/_view so its different to the existing browser settings. We want this as an additional thing separate from the existing "Interface" which contains the existing browser settings.